scripts/jira/scrum.py

```diff
@@ -0,0 +1,420 @@
+# Bugs waiting code review story points at the end of a sprint
+CSV_FIELD_BUG_WAITING_CODE_REVIEW = 'Bugs waiting code review story points'
+# Bugs in progress story points at the end of a sprint
+CSV_FIELD_BUG_IN_PROGRESS = 'Bugs in progress story points'
+# Bugs not started story points at the end of a sprint
+CSV_FIELD_BUG_NOT_STARTED = 'Bugs not started story points'
+# Total number of bugs / total number of issues
+CSV_FIELD_DEFECT_DENSITY_RATIO = 'Defect density ratio'
+# Resolved points / planned points ratio
+CSV_FIELD_PREDICTABILITY_RATIO = 'Predictability ratio'
+# Total story points / planned story points * 100
+CSV_FIELD_SCOPE_CHANGE_PERCENTAGE = 'Scope change percentage'
+# Goals achieved / goals planned * 100
+CSV_FIELD_GOAL_COMPLETION_PERCENTAGE = 'Goal completion percentage'
+# Working days in a sprint
+CSV_FIELD_WORKING_DAYS = 'Working days'
+# Planned story points / working days
+CSV_FIELD_PLANNED_SP_PER_DAY = 'Planned story points per working day'
+# Total story points / working days
+CSV_FIELD_TOTAL_SP_PER_DAY = 'Total story points per working day'
+# Resolved story points / working days
+CSV_FIELD_RESOLVED_SP_PER_DAY = 'Velocity per working day'
+# Proportion of time work was actively progressing versus waiting. resolved / (resolved + in progress + in code review + in qe) * 100
+CSV_FIELD_FLOW_EFFICIENCY = 'Flow efficiency'
+# Proportion of time work was actively progressing versus waiting. resolved bugs / (resolved bugs + in progress bugs + in code review bugs + in qe bugs) * 100
+CSV_FIELD_FLOW_EFFICIENCY_BUGS = 'Bug flow efficiency'
+# Proportion of work that remains incomplete at the end of the sprint. (in progress + in code review + in qe + not started) / planned * 100
+CSV_FIELD_WIP_RATIO = 'WIP ratio'
+# How drastically the sprint scope changes due to story points being added or removed after the sprint has started
+# (added + removed) / planned * 100
+CSV_FIELD_SCOPE_VOLATILITY = 'Scope volatility'
+# How many of the bugs were resolved vs planned/added. resolved bugs / total bugs * 100
+CSV_FIELD_DEFECT_REMOVAL_EFFICIENCY = 'Defect removal efficiency'
+
+CSV_HEADERS = [
+    CSV_FIELD_SPRINT_NAME,
+    CSV_FIELD_PLANNED_TOTAL,
+    CSV_FIELD_RESOLVED_TOTAL,
+    CSV_FIELD_NOT_STARTED_TOTAL,
+    CSV_FIELD_IN_PROGRESS_TOTAL,
+    CSV_FIELD_CODE_REVIEW_TOTAL,
+    CSV_FIELD_BUG_TOTAL,
+    CSV_FIELD_BUG_RESOLVED,
+    CSV_FIELD_BUG_WAITING_QE,
+    CSV_FIELD_BUG_WAITING_CODE_REVIEW,
+    CSV_FIELD_BUG_IN_PROGRESS,
+    CSV_FIELD_BUG_NOT_STARTED,
+    CSV_FIELD_WORKING_DAYS,
+    CSV_FIELD_PLANNED_SP_PER_DAY,
+    CSV_FIELD_TOTAL_SP_PER_DAY,
+    CSV_FIELD_RESOLVED_SP_PER_DAY,
+    CSV_FIELD_GOAL_COMPLETION_PERCENTAGE,
+    CSV_FIELD_SCOPE_CHANGE_PERCENTAGE,
+    CSV_FIELD_PREDICTABILITY_RATIO,
+    CSV_FIELD_FLOW_EFFICIENCY,
+    CSV_FIELD_WIP_RATIO,
+    CSV_FIELD_SCOPE_VOLATILITY,
+    CSV_FIELD_DEFECT_DENSITY_RATIO,
+    CSV_FIELD_DEFECT_REMOVAL_EFFICIENCY,
+    CSV_FIELD_FLOW_EFFICIENCY_BUGS,
+]
+
+AI_PROMPTS = [
```

Comment:
Do we want to add relevant text to the readme file?

Reply:
Done